estuary_updater/handlers/errata.py

```diff
@@ -65,7 +71,10 @@ def activity_status_handler(self, msg):
         advisory_type = msg['body']['headers']['type'].lower()
         advisory_info = advisory_json['errata'][advisory_type]
 
-        embargoed = msg['body']['headers']['errata_status'] == 'REDACTED'
```

Comment:
Both message formats share the key `synopsis` and that will be equal to `REDACTED` if the advisory is embargoed.